lib/reversetunnel/api.go

```diff
@@ -83,15 +89,15 @@ func (params DialParams) String() string {
 // There are two implementations of this interface: local and remote sites.
 type RemoteSite interface {
 	// DialAuthServer returns a net.Conn to the Auth Server of a site.
-	DialAuthServer() (net.Conn, error)
+	DialAuthServer(DialParams) (net.Conn, bool, error)
```

Comment:
We should either name these returns, or provide an explanation in the comment as for what the boolean indicates.

Reply:
Added both.

Reply:
Thank you - that's perfect.